css-fonts-4/Overview.bs

```diff
@@ -8332,12 +8332,12 @@ Changes
 	<li>Clarified the font shorthand, using the terms set explicitly, reset implicitly and cascade implicitly</li>
 	<li>Clarified shorter representation for zero ranged descriptors</li>
 	<li>Tightened definition of first available font</li>
-	<li>Added emoji as a keyword to unicode‑range</li>
```

Comment:
What changed here?

Reply:
The dash was actually a different character code, VSCode was flagging it, so I swapped it for a normal short dash

Reply:
Okay I thought that might be the case, the standard hyphen-minus is indeed correct.